tests/acceptance/10_ncf_internals/unsafe/modules/zypper_pattern_module.cf

```diff
@@ -34,11 +34,14 @@ bundle agent init
 {
   # Build packages + a local repo + remove packages used in test
   vars:
-    "repo_path" string => "/usr/share/ncf/tests/acceptance/30_generic_methods/modules/repo_rudder_tests";
+    "pwd" string => dirname("${this.promise_filename}");
+    "repo_path" string => "${pwd}/repo_rudder_tests";
+    "ncf_path" string => "${pwd}/../../../../..";
```

Comment:
we should probably make it a global variable in a library :)

Reply:
We actually have one in the tests, `ncf_tree`